src/str_extract.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os, sys, re
+import argparse
+
+def strextract(dir, suffix=None, path=False, all=False):  
+    # Expression régulière pour trouver des chaînes littérales entre guillemets simples ou doubles
+    # compiled_re = re.compile(r"^.*'.*'.*"'"'r".*"'"'r"$", re.IGNORECASE)
+    regex_chain = re.compile(r'(["\'])(.*?)\1', re.IGNORECASE)
+    # Parcourir récursivement le répertoire
+    for root, directorys, filename in os.walk(dir):
+        for name_file in filename:
```

Comment:
Rename `name_file` into `filename` (`name_file` would be a file, not a name)

Reply:
Ok, 
And what if i'd name it `name_of_the_file` so that it doesn't itch my eyes on top of burning my brain ?

Reply:
OK